app/utils/estimate_prioritization/estimate_prioritization.py

```diff
@@ -121,6 +123,28 @@ def get_prioritized_estimates(estimates: pd.DataFrame,
             selected_estimates.append(get_pooled_estimate(study_estimates))
         else:
             selected_estimates.append(study_estimates)
-    
-    selected_estimate_df = pd.concat(selected_estimates, axis = 1).T.astype(estimates.dtypes.to_dict())
-    return selected_estimate_df
\ No newline at end of file
+
+    if pool:
```

Comment:
The other way to do this would be to group by the thing you don't want to pool by as part of the estimate prioritization step. You'd need to pass in a second variable to group by:

for study_name, study_estimates in estimates.groupby('study_name'):
  for subgroup_name, subgroup_estimates in subgroup_estimates.groupby(subgroup_var):

and, in this case, subgroup_var could be something like province

tbh, I think this is a bit of a worse solution, because to implement this for geography would require using a subgroup var that is concatenated country and city and state...

yeah this is a bad idea i should go to bed but imma just leave this comment anyway